iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/ConnectableConnection.java

```diff
@@ -76,6 +76,6 @@ public void apply(Network network, NamingStrategy namingStrategy, boolean throwE
         } else {
             LOG.info("Connectable {} has NOT been connected.", connectableId);
         }
-        connectableConnectionReport(reportNode, connectable, hasBeenConnected);
+        connectableConnectionReport(reportNode, connectable, hasBeenConnected, side);
```

Comment:
As for `AbstractDisconnection`, could you also add the side in the logger message?

Reply:
done